lib/attributor/types/tempfile.rb

```diff
@@ -31,6 +31,9 @@ def self.load(value,context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
       super
     end
 
+    def self.family
+      String.family
```

Comment:
I went with the representation here too, since the `dump` just uses the path to the file.